icpc-wf/ansible/roles/domserver/templates/nginx-domjudge-inner.j2

```diff
@@ -0,0 +1,43 @@
+# inner nginx configuration for DOMjudge
```

Comment:
Shouldn't this file be set from the domjudge repo?

Reply:
I looked at that, but the old ansible code was changing so much of the original config, that I think it doesn't make sense. The regexp replaces also made that the ansible tasks were always marked as "changed".

Reply:
I indeed think this is way easier. We just have to keep it 'in sync' if we change stuff in the main repo